helix-rest/src/test/java/org/apache/helix/rest/server/AbstractTestClass.java

```diff
@@ -451,7 +451,7 @@ protected ClusterControllerManager startController(String cluster) {
     for (int i = 0; i < numJobs; i++) {
       JobConfig.Builder job =
           new JobConfig.Builder().setCommand("DummyCommand").setTargetResource("RESOURCE")
-              .setWorkflow(workflowName);
+              .setWorkflow(workflowName).setJobId(workflowName + "_" + JOB_PREFIX + i);
```

Comment:
This is necessary to have due to #1391. Previously we were relying on the ideal state. Now once the config is added, the controller will process the job. If the job config does not have an ID field, TaskData cache does not see the config and the job will not get processed.